app/models/post.rb

```diff
@@ -0,0 +1,9 @@
+class Post < ApplicationRecord
+  belongs_to :user, optional: true
+  has_one :picture, as: :picturable
+
+  enum status: { public: 1, private: 2 }, _prefix: true
+
+  validates :body, length: { in: 2..1000 }
+  validates :likes, numericality: { only_integer: true }
```

Comment:
https://trello.com/c/cZx4rR4Z/26-dev-create-like
Likes should be a model so we can associate likes with users